rssfeed/templates/rssfeed/inclusion_tags/entry_list.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+{% if entry_list %}
+    {% for entry in entry_list %}
+        <div class="feed_entry">
+            <img src="{{ entry.image }}" height="200" width="300">
+            <h3><a href="{{ entry.link }}">{{ entry.title|safe }}</a></h3>
+            <p class="rssfeed_subtitle">
```

Comment:
Use a blocktrans to translate a group of related terms containing variables in between them. Not all languages follow the same grammatical structure and blocktrans allows the translator to move phrases around.